src/main/java/com/sogeti/SoSenseiPsyche/model/CodeGenerator.java

```diff
@@ -0,0 +1,4 @@
+package com.sogeti.SoSenseiPsyche.model;
+
+public class CodeGenerator {
```

Comment:
Remove empty classes if not used. Also code generator is finished in another PR. So this will be removed

Reply:
done